### PR TITLE
Blog post for Prague conf launch and smaller tweaks

### PR DESCRIPTION
**Pull request by thatdocslady**
Title says it all :) 

TODO:
- [x] @plaindocs remove RH from the sponsor list
- [x] @ericholscher review blog post
- [x] @plaindocs CFP form

### CODE REVIEW

**Comment by plaindocs** [comment]
~@thatdocslady did you want to move the bullet point lists into the intial blog post? or leave as is?~

**Comment by thatdocslady** [comment]
@plaindocs which bullet point list?

**Comment by plaindocs** [comment]
This is mergeable as far as I'm concerned.

File: docs/conf/prague/2018/news/prague-18-welcome.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+-------
+
+Ticket prices and descriptions are available on our `Tickets <http://www.writethedocs.org/conf/prague/2018/tickets/>`_ page. Last year we sold out at 250 attendees, and we're going to maintain this maximum capacity again this year, to help our attendees feel comfortable in this beautiful venue.
+
+We believe that everyone who cares about this topic has a place at our event, and we keep ticket prices low so that it is accessible to the widest range of people possible.
+However, if you can't afford these prices and still wish to attend, please email us at prague@writethedocs.org, and we can work on getting you reduced price tickets or volunteer options.
+
+As with the previous years, we have tiered pricing to make it fair for everyone.
+Please buy the ticket that best fits your situation.
+Tickets are `on sale now <http://www.writethedocs.org/conf/eu/2017/tickets/>`_.
```
Comment (ericholscher) [inline docs/conf/prague/2018/news/prague-18-welcome.rst]: Link is to last year.

**Review by ericholscher — APPROVED** [review]
Ship it with small fix.

**Pull request merged**